<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>451</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SandboxiePlus Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabs">
       <property name="font">
        <font>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabGeneral">
        <attribute name="title">
         <string>General Options</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="1" column="1">
            <widget class="QCheckBox" name="chkDarkTheme">
             <property name="text">
              <string>Use Dark Theme</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QCheckBox" name="chkShowTray">
             <property name="text">
              <string>Show Sys-Tray</string>
             </property>
            </widget>
           </item>
           <item row="9" column="2">
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Tray options</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Restart required (!)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="chkAutoStart">
             <property name="text">
              <string>Start with Windows</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>On main window close:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="9" column="1">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="8" column="1">
            <widget class="QComboBox" name="onClose"/>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="uiLang"/>
           </item>
           <item row="5" column="1">
            <widget class="QCheckBox" name="chkWatchConfig">
             <property name="text">
              <string>Watch Sandboxie.ini for changes</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="2">
            <widget class="QCheckBox" name="chkNotifications">
             <property name="text">
              <string>Show Notifications for relevant log Messages</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QCheckBox" name="chkShellMenu">
             <property name="text">
              <string>Add 'Run Sandboxed' to the explorer context menu</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabAdvanced">
        <attribute name="title">
         <string>Advanced Options</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="8" column="1" colspan="2">
            <widget class="QCheckBox" name="chkAdminOnlyFP">
             <property name="text">
              <string>Only Administrator user accounts can use Disable Forced Programs command</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QCheckBox" name="chkAdminOnly">
             <property name="text">
              <string>Only Administrator user accounts can make changes</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_44">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Config protection</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QCheckBox" name="chkPassRequired">
             <property name="text">
              <string>Password must be entered in order to make changes</string>
             </property>
            </widget>
           </item>
           <item row="7" column="2">
            <widget class="QPushButton" name="btnSetPassword">
             <property name="text">
              <string>Change Password</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="fileRoot"/>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QLineEdit" name="regRoot"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_43">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Sandbox default</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="10" column="2">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Sandbox file system root: </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Sandbox ipc root: </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="2">
            <widget class="QLineEdit" name="ipcRoot"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Sandbox registry root: </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QCheckBox" name="chkSeparateUserFolders">
             <property name="text">
              <string>Separate user folders</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="9" column="1">
            <widget class="QCheckBox" name="chkClearPass">
             <property name="text">
              <string>Clear password when main window becomes hidden</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabAlerts">
        <attribute name="title">
         <string>Program Alerts</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0" rowspan="3">
            <widget class="QTreeWidget" name="treeWarnProgs">
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btnAddWarnProg">
             <property name="text">
              <string>Add Program</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="btnDelWarnProg">
             <property name="text">
              <string>Remove Program</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>When any of following programs is launched outside any sandbox, Sandboxie will issue message SBIE1301.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabCompat">
        <attribute name="title">
         <string>Software Compatybility</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="4" column="0">
            <widget class="QCheckBox" name="chkNoCompat">
             <property name="text">
              <string>In the future, don't check software compatibility</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btnAddCompat">
             <property name="text">
              <string>Enable</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="btnDelCompat">
             <property name="text">
              <string>Disable</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" rowspan="3">
            <widget class="QTreeWidget" name="treeCompat">
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="2" column="1">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Sandboxie has detected the following software applications in your system. Click OK to apply configuration settings which will improve compatibility with these applications. These configuration settings will have effect in all existing sandboxes and in any new sandboxes.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
